### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by CThierrin**
Resolves Issue #3

### CODE REVIEW

**Comment by dchiller** [comment]
In future, you can link a PR to an issue by [using a keyword](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword). 

So if say somewhere in the comment "Resolves #3" or "Closes #3" or "Fixes #3" or some such that will link the PR to the issue (you can also do this manually in the "Development" panel to the right") and then the issue will close when the PR is successfully merged.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+**MEI 3 to 5**
+=====================================
+
+This repository contains a collection of scripts designed to update MEI 3 files to MEI 5. The current implementation of this repository only supports upgrading MEI files from the Liber Usualis. Future developments may extend this functionality to support additional MEI files.
+
+
+**Prerequisites**
+--------------------------------
+
+The only required dependency is the lxml library. You can i
```
Comment (dchiller) [inline README.md]: Looks like there is a missing phrase here...

File: README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+**Usage**
+-----
+
+1. If you do not have it already, install lxml using `pip install lxml`
```
Comment (dchiller) [inline README.md]: Provide a version of `lxml`

File: README.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+2. Download the scripts in this repository into the same directory as your MEI files
```
Comment (dchiller) [inline README.md]: I think better would be to have people clone this repository locally and then put the MEI files into that directory (or move the scripts into the same directory as files post-cloning).

File: README.md (hunk 1 above)
Comment (CThierrin) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (CThierrin) [reply README.md]: Fixed

File: README.md (hunk 3 above)
Comment (CThierrin) [reply README.md]: Fixed

File: README.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+**MEI 3 to 5**
+=====================================
+
+This repository contains a collection of scripts designed to update MEI 3 files to MEI 5. The current implementation of this repository only supports upgrading MEI files from the Liber Usualis. Future developments may extend this functionality to support additional MEI files.
+
+**Usage**
+-----
+
+1. If you do not have it already, install lxml 5.0 using `pip install lxml`
```
Comment (dchiller) [inline README.md]:
```suggestion
1. If you do not have it already, install lxml 5.0 using `pip install lxml==5.0`
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+2. Clone the repository and download the scripts in this repository into the same directory as your MEI files
```
Comment (dchiller) [inline README.md]:
```suggestion
2. Clone the repository and move the scripts in this repository into the same directory as your MEI files
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+3. Run the `liberbatch.py` script
```
Comment (dchiller) [inline README.md:11]: Looking at `liberbatch.py`: won't this look in `C:\Users\cole_\Downloads\...` on everyone's machine, which probably won't exist except for yours? 

This point is out of scope of this PR, so I've made another issue.

**Review by dchiller — APPROVED** [review]
